TravelAgent/TravelAgent/Data/Repositories/Interfaces/IEmployeeTravelRepository.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using TravelAgent.Data.Entities;
+
+namespace TravelAgent.Data.Repositories.Interfaces
+{
+    interface IEmployeeTravelRepository : IRepository<EmployeeTravel>
+    {
+    }
```

Comment:
What's the purpose of these empty interfaces?

Reply:
Add specific method